test/Microsoft.Azure.WebJobs.Host.EndToEndTests/WebJobs.Host.EndToEndTests.csproj

```diff
@@ -29,15 +29,16 @@
     <PackageReference Include="StyleCop.Analyzers" Version="1.1.0-beta004">
       <PrivateAssets>all</PrivateAssets>
     </PackageReference>
+    <PackageReference Include="Microsoft.Extensions.Azure" Version="1.1.0-beta.2" />
     <PackageReference Include="Microsoft.Extensions.Logging" Version="2.2.0" />
     <PackageReference Include="Microsoft.NET.Test.Sdk" Version="15.5.0" />
     <PackageReference Include="Moq" Version="4.7.145" />
     <PackageReference Include="xunit" Version="2.3.1" />
     <PackageReference Include="xunit.runner.visualstudio" Version="2.3.1" />
+    <PackageReference Include="Microsoft.Azure.WebJobs.Extensions.Storage" Version="5.0.0-beta.5" />
```

Comment:
So previously both Extensions.Storage and Host.Storage were via project references, and the versions used were the ones being built. Now we're pulling in the external dependency for Extensions.Storage, but still referencing our own Host.Storage. What version are we testing then?

Reply:
The tests in EndToEndTests and FunctionalTests project should be doing testing related to integrating those new T2 extensions with WebJobs. All the duplicate tests that have been ported to the Azure SDK repo are removed.